adafruit_minimqtt/adafruit_minimqtt.py

```diff
@@ -300,8 +303,7 @@ def connect(self, clean_session=True):
                 raise MMQTTException("Invalid broker address defined.", e)
 
         # Fixed Header
-        fixed_header = bytearray()
-        fixed_header.append(0x10)
+        fixed_header = bytearray([0x10])
```

Comment:
Could we do this in a separate PR instead? I like it, but it shouldn't be included with this PR.

Reply:
ah, you caught me... good eye. Opened a new PR with this change